.github/workflows/ci.yml

```diff
@@ -9,7 +9,7 @@ jobs:
       fail-fast: false
       matrix:
         version:
-          - '1.10'
+          - '1.10.8'
```

Comment:
if you leave that `1.10` it will automatically choose the latest patch release

Reply:
ah ok